GameData/RealismOverhaul/RO_SuggestedMods/Squad/RO_Squad_Utility.cfg

```diff
@@ -439,6 +439,60 @@
 		@resourceAmount = 0.015
 	}
 }
+@PART[spotLight1_v2]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@mass = 0.0035
+	@MODULE[ModuleLight]
+	{
+		@resourceAmount = 0.03
+	}
+}
+@PART[spotLight2_v2]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@mass = 0.0035
+	@MODULE[ModuleLight]
+	{
+		@resourceAmount = 0.015
+	}
+}
+@PART[domeLight1]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@mass = 0.001
+	@MODULE[ModuleLight]
+	{
+		@resourceAmount = 0.015
+	}
+}
+@PART[stripLight1]:FOR[RealismOverhaul]
```

Comment:
Similarly for here... if you're applying the exact same changes, prefer to see it in a single patch rather than duplicated patches.  Ditto for `navLight1` + `spotLight3`